ldk/goja/src/whisper/index.ts

```diff
@@ -0,0 +1,255 @@
+export enum WhisperComponentType {
```

Comment:
This could potentially be a card in the future- add more typedoc comments for these to give users more context as to what these properties do within OliveHelps.

Reply:
We don't currently have documentation with this enums in, but it's a good point